examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>There is already a function in devito called <code>plot_velocity</code>.</p><p>https://github.com/devitocodes/devito/blob/2b434643d0ecfc70e9c5bb1277081d425136c9de/examples/seismic/plotting.py#L50</p><p>Can that work?</p><p>Other notebooks use that as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
I know that there are functions to plot the values of receivers and displacament, but these functions using the model structure and in these notebooks we don't use these structure.